packages/code-du-travail-frontend/src/modules/common/Share.tsx

```diff
@@ -23,7 +28,7 @@ export const Share = ({ title, metaDescription }: Props): JSX.Element => {
   };
 
   return (
-    <div className={fr.cx("fr-follow__social")}>
+    <div className={`${className} ${fr.cx("fr-follow__social")}`}>
```

Comment:
```suggestion
    <div className={`${fr.cx("fr-follow__social")} ${className} `}>
```

pour virer l'espace au début

Reply:
d'ailleurs, je valide l'approche du `className`, car c'est un composant partagé. dans le cas où c'est local au feature je pense pas que ça vaille coup de faire ça